src/Sarif.UnitTests/SarifTransformerTests.cs

```diff
@@ -194,7 +189,10 @@ public void SarifTransformerTests_ToCurrent_OneRunWithFiles()
                           ]
                         },
                         ""file:///home/buildAgent/bin/app.zip"": {
-                          ""mimeType"": ""application/zip""
+                          ""mimeType"": ""application/zip"",
+                          ""properties"": {
+                            ""my_key"": ""some value""
+                          }
```

Comment:
But now you need to test that the property bag was copied.

Reply:
Lines 243-44?